functions/src/calculator/index.js

```diff
@@ -0,0 +1,47 @@
+import * as functions from 'firebase-functions'
+import * as admin from 'firebase-admin'
+import calculateResult from '../utils/calculateResult'
+import { to } from '../utils/async'
+
+/**
+ * @param req - Express HTTP Request
+ * @param res - Express HTTP Response
+ */
+async function calculatorRequest(req, res) {
+  const number1 = Number(req.body.number1)
```

Comment:
It may be good to check for the object before getting the parameters off of it to prevent errors. It could be done using something [like `get` from lodash](https://lodash.com/docs/4.17.10#get) or destructuring with a default:

```js
const {
  number1: unparsedNumber1, 
  number2: unparsedNumber2
} = req.body || {}
const number1 = Number(unparsedNumber1)
const number2 = Number(unparsedNumber2)
```
or
```js
import { get } from 'lodash'

const number1 = Number(get(req, 'body.number1'))
const number2 = Number(get(req, 'body.number2'))
```

This also brings up the point: Do we want it to assume 0 if no number is passed or return an error response to the user?